phala-cloud/confidential-ai/confidential-gpu/overview.mdx

```diff
@@ -0,0 +1,44 @@
+---
+description: Rent Confidential GPU on Phala Cloud with various GPU device and flexible subscription.
+title: Confidential GPU
+---
+
+## Overview
```

Comment:
Just remove this line. It's ok to put overview paragraph at the beginning without a subtitle